code/modules/admin/sql_message_system.dm

```diff
@@ -354,12 +411,35 @@
 						alpha = 10
 						skipped = TRUE
 					alphatext = "filter: alpha(opacity=[alpha]); opacity: [alpha/100];"
-
-			var/list/data = list("<p style='margin:0px;[alphatext]'> <b>[timestamp] | [server] | [admin_key]")
+			var/list/data = list("")
+			if(!style_defined)
+				data +="<style>"
+				data +="a {color: #0000EE; font-weight: normal !important}"
+				data +="#hii {margin:0px; font-size: 18px; padding: 0px 0px 0px 10px; border-radius: 25px; border: 1px solid black; background: #cc3300; color: white;[alphatext] }"
+				data +="#med {margin:0px; font-size: 18px; padding: 0px 0px 0px 10px; border-radius: 25px; border: 1px solid; background: #ff6600;[alphatext]}"
+				data +="#low {margin:0px; font-size: 18px; padding: 0px 0px 0px 10px; border-radius: 25px; border: 1px solid; background: #FFCC00;[alphatext]}"
+				data +="#non {margin:0px; font-size: 18px; padding: 0px 0px 0px 10px; border-radius: 25px; border: 1px solid; background: #339900;[alphatext]}"
+				data +="#naa {margin:0px; font-size: 18px; padding: 0px 0px 0px 10px;[alphatext]}"
```

Comment:
I think you should be able to do inherited css classes so you don't need to rewrite this styling for each to only change the colors.
Something like 
```
#severity {margin:0px; font-size: 18px; padding: 0px 0px 0px 10px; border-radius: 25px; border: 1px solid black;}
#severity.high {background: #cc3300; color: white;[alphatext])
#severity.med {background: #ff6600}
```
etc.
But test it out because I'm not confident in my css.

Reply:
actually this messes up notes fading away, I just realized. Hold on.